projects/plugins/protect/src/js/components/footer/index.jsx

```diff
@@ -21,8 +21,29 @@ import { useProductCheckoutWorkflow } from '@automattic/jetpack-connection';
 import styles from './styles.module.scss';
 import useAnalyticsTracks from '../../hooks/use-analytics-tracks';
 import { SECURITY_BUNDLE } from '../admin-page';
+import useProtectData from '../../hooks/use-protect-data';
+
+const ProductPromotion = ( { onSecurityAdd, hasCheckoutStarted, hasSecurityBundle } ) => {
+	if ( ! hasSecurityBundle ) {
+		return (
+			<div className={ styles.section }>
+				<Title>
+					{ __( 'Increase your site protection with Jetpack Scan', 'jetpack-protect' ) }
+				</Title>
+				<Text className={ styles.paragraphs }>
+					{ __(
+						'With your Jetpack Security bundle you have access to Jetpack Scan. Automatically scan your site from the Cloud, get email notifications and perform one-click fixes.',
+						'jetpack-protect'
+					) }
+				</Text>
+
+				<Button variant="external-link" weight="regular" href="#">
```

Comment:
Just add a link to our redirect service with a new source and we figure out later where to point it to. (I think this points to Calypso green Scan page)

Something like `protect-footer-get-started-scan`

Reply:
https://github.com/Automattic/jetpack/pull/24262/commits/2f106bae43799da0d0e27fbb4377ee0c507abf26